packages/askar/src/utils/askarWalletConfig.ts

```diff
@@ -0,0 +1,42 @@
+import type { WalletConfig } from '@aries-framework/core'
+
+import { KeyDerivationMethod, WalletError } from '@aries-framework/core'
+import { StoreKeyMethod } from 'aries-askar-test-shared'
+
+export const keyDerivationMethodToStoreKeyMethod = (keyDerivationMethod?: KeyDerivationMethod) => {
+  if (!keyDerivationMethod) {
+    return undefined
+  }
+
+  const correspondanceTable = {
+    [KeyDerivationMethod.Raw]: StoreKeyMethod.Raw,
+    [KeyDerivationMethod.Argon2IInt]: `${StoreKeyMethod.Kdf}:argon2i:int`,
+    [KeyDerivationMethod.Argon2IMod]: `${StoreKeyMethod.Kdf}:argon2i:mod`,
+  }
+
+  return correspondanceTable[keyDerivationMethod] as StoreKeyMethod
+}
+
+export const uriFromWalletConfig = (walletConfig: WalletConfig, basePath: string): { uri: string; path?: string } => {
```

Comment:
We may need to tweak the wallet config a bit if that leads to better results. I've even been thinking about removing the wallet config from the agent config and just making it specific to the wallet interface so there's no need for a standardized wallet config interface. However that would cause some issues for e.g. the tenant module (that needs to create wallet configs on demand).  Thoughts?

Reply:
I agree that it may become difficult to define a single base interface for wallets. If it's getting complicated to get a common interface for two projects originated from the same codebase (sort of), I can imagine that will be very hard to adapt any kind of storage backend to AFJ. 

So I think it will be nice if we can leave the format to the nature of the wallet itself and make it free to choose how to get configured. We'll check how to allow Tenants module to support it (maybe a callback for dynamic config generation?).